api_app/analyzers_manager/serializers.py

```diff
@@ -0,0 +1,127 @@
+from rest_framework import serializers as rfs
+from django.conf import settings
+
+import os
+import json
+import logging
+
+from .models import AnalyzerReport
+from . import helpers
+
+from intel_owl.secrets import get_secret
+
+
+logger = logging.getLogger(__name__)
+
+
+class BaseField(rfs.Field):
+    def to_representation(self, value):
+        return value
+
+    def to_internal_value(self, data):
+        return data
+
+
+class AnalyzerReportSerializer(rfs.ModelSerializer):
+    class Meta:
+        model = AnalyzerReport
+        fields = "__all__"
+
+
+class AnalyzerConfigSerializer(rfs.Serializer):
+    name = rfs.CharField(required=True)
+    type_ = rfs.CharField(required=True)
```

Comment:
should be just `type`, I assume your IDE must have given a warning because `type` is an inbuilt keyword but it should still work.

Reply:
Yeah it did. Just thought I would avoid reserved keywords so did that